src/model/mosaic/MosaicFlags.ts

```diff
@@ -69,7 +74,7 @@ export class MosaicFlags {
      * @returns {number}
      */
     public getValue(): number {
-        return (this.supplyMutable ? 1 : 0) + (this.transferable ? 2 : 0) + (this.restrictable ? 4 : 0);
+        return (this.supplyMutable ? 1 : 0) + (this.transferable ? 2 : 0) + (this.restrictable ? 4 : 0) + (this.revokable ? 8 : 0);
```

Comment:
consider refactoring since you have same summation on line 68

Reply:
Refactored, created common method